### ISSUE TITLE
Further dev in step02: Train posture recognition cnn

### ISSUE THREAD

**Issue by YanzhenHuang** [FYP-hyz-mjj-2024/posture_mmpose#19]
- [x] Normalization
- Per-channel normalize mean & std during application.
- Add per-channel batch norm while training.

- [ ] Combine multiple loss functions.

**Comment by YanzhenHuang** [comment]
Example: Per-channel batch norm while training.
```python

class MLP3d(nn.Module):
    def __init__(self, input_channel_num, output_class_num):
        super(MLP3d, self).__init__()
        self.k = (3, 5, 5)  # kernel_size
        self.m_k = 2  # max_pool kernel_size
        self.activation = nn.ELU()

        self.conv_layers = nn.Sequential(
            nn.Conv3d(in_channels=input_channel_num, out_channels=8, kernel_size=self.k, padding='same'),
            nn.BatchNorm3d(num_features=8),  # Batch norm
            self.activation,
            
            nn.Conv3d(in_channels=8, out_channels=16, kernel_size=self.k, padding='same'),
            nn.BatchNorm3d(num_features=16),  # Batch norm
            self.activation,

            nn.Conv3d(in_channels=16, out_channels=32, kernel_size=self.k, padding='same'),
            nn.BatchNorm3d(num_features=32),  # Batch norm
            self.activation,
            
            nn.Conv3d(in_channels=32, out_channels=32, kernel_size=self.k, padding='same'),
            nn.BatchNorm3d(num_features=32),  # Batch norm
            self.activation,
            
            nn.MaxPool3d(kernel_size=self.m_k, stride=self.m_k),
        )

        self.fc_layers = nn.Sequential(
            nn.Linear(in_features=2880, out_features=256),
            self.activation,
            nn.Linear(in_features=256, out_features=output_class_num)
        )

    def forward(self, x):
        x = self.conv_layers(x)
        x = x.view(x.size(0), -1)  # 192
        x = self.fc_layers(x)

        return x
```
